tools/build-utils.js

```diff
@@ -1,66 +1,74 @@
 "use strict";
 
 const fs = require("fs");
-const child_process = require("child_process");
-const esbuild = require('esbuild');
+const path = require("path");
+const oxc = require("oxc-transform");
+const sucrase = require("sucrase");
+const fg = require("fast-glob");
+const { execSync } = require("child_process");
 
-const copyOverDataJSON = (file = 'data') => {
-	const files = fs.readdirSync(file);
-	for (const f of files) {
-		if (fs.statSync(`${file}/${f}`).isDirectory()) {
-			copyOverDataJSON(`${file}/${f}`);
-		} else if (f.endsWith('.json')) {
-			fs.copyFileSync(`${file}/${f}`, require('path').resolve('dist', `${file}/${f}`));
-		}
+function copyJSON(dir = 'data') {
+	for (const src of fg.sync(`${dir}/**/*.json`)) {
```

Comment:
I would be very surprised if this made enough of a difference to be worth grabbing the fast-glob dependency...